### ISSUE TITLE
Deploy Access the Data on dev environment

### ISSUE THREAD

**Issue by JasonEb** [hackforla/incubator#14]
### Dependency
~When the Terraform setup instructions is ready~

### Overview
The Access The Data team seeks to deploy their app within HackForLA's AWS incubator account. @gjagnoor has a docker build ready for a `dev` environment. 

### Action Items
- [ ] Setup project.hcls
- [ ] Apply Terragrunt to provision AWS
- [ ] Push Docker-build onto provisioned Fargate
- [ ] Setup github action to deploy back-end/front-end builds to production environment

### Dependencies
- [x] https://github.com/hackforla/ops/issues/81

### Resources/Instructions
[Incubator](https://drive.google.com/drive/folders/1_UZtr0P-GYR8aQbNiEGwS5_vdDccooSG)
[CivicTechJobs Repo](https://github.com/hackforla/CivicTechJobs)

**Comment by nyarly** [comment]
@JasonEb We're eager to get Access-the-data up in staging. Could I get added as an assignee for this issue?

I suspect I'll also need to be added (at least temporarily) to the ops group in order to work at all with `terragrunt` as well.

**Comment by nyarly** [comment]
I've made some progress here - need to find some time to test it out

**Comment by nyarly** [comment]
Working `terraform plan` for AtD dev environment.

Next steps: 
* build images based on the project's Docker Compose setup
* `terraform apply` for dev
* Push images
* Test & debug

**Comment by mxajPrice** [comment]
I am messaging Bonnie to have accessthedata.org point to nameservers:

- ns-365.awsdns-45.com
- ns-1756.awsdns-27.co.uk
- ns-837.awsdns-40.net
- ns-1109.awsdns-10.org

**Comment by ExperimentsInHonesty** [comment]
Name servers have been changed. The following messages was received

> Success
> These DNS settings are active. Changes are published immediately, but may take time to propagate

**Comment by nyarly** [comment]
I have a deploy applied for this. I'll need to do a little verification and debug once we have the NS set up, but we're close closing this issue

**Comment by nyarly** [comment]
Fixing a Terraform issue required recreating the Zone file, which changed the nameserver records.

Bonnie, could you please update the accessthedata.org domain at its registrar (SquareSpace, IIUC) to be:
```
ns-234.awsdns-29.com
ns-876.awsdns-45.net
ns-1680.awsdns-18.co.uk
ns-1125.awsdns-12.org
```

Thank you!

**Comment by ExperimentsInHonesty** [comment]
Name servers have been updated

**Comment by nyarly** [comment]
Progress debugging the deployment. Needed extra permissions to access DB passwords in parameters, and logging configurations. Now investigating issues that the running application is having - I think the environment variables for the database configuration are not aligned with what the application expects. I'm investigating the options there.

**Comment by nyarly** [comment]
Database is connecting! Now, I need to figure out how to connect the individual tasks via network (this is documentation reading exercise). It's possible once this is complete, AtD will be deployed.

**Comment by robinglov** [comment]
Update from @nyarly : a developer who is strong in Python and understands how to containerize the application is required to take this on.

**Comment by ExperimentsInHonesty** [comment]
Re write issue so that it has current instructions, and clean up the issue. or make a new issue and reference this issue as legacy.

**Comment by ale210** [comment]
stale, current container doesn't start on old incubator
